chia/wallet/wallet_state_manager.py

```diff
@@ -529,13 +528,37 @@ async def synced(self):
             return True
         return False
 
-    def set_sync_mode(self, mode: bool, sync_height: uint32 = uint32(0)):
-        """
-        Sets the sync mode. This changes the behavior of the wallet node.
-        """
-        self.sync_mode = mode
-        self.sync_target = sync_height
-        self.state_changed("sync_changed")
+    @property
+    def sync_mode(self) -> bool:
+        return self._sync_target is not None
+
+    @property
+    def sync_target(self) -> Optional[uint32]:
+        return self._sync_target
```

Comment:
the previous member `sync_target` is not an `Optional`, there's no risk for issues changing it into an optional?

Reply:
It wasn't used in any place so i don't think there is a risk.

Reply:
oh, so we could just use a `bool` then?

Reply:
We could, the reason i picked the height still was because im thinking about adding it in some follow up PR to the sync status in the RPC endpoint and since and since i anyway wanted to pass the height into the context to be able to log it i just did it like this now. Want to change it rather? I don't mind.